### DISCUSSION TITLE
Question: Text Summarization

### DISCUSSION THREAD

**Discussion by paul-bell** [winkjs/wink-nlp#51] [New Features & Ideas]
Hello again, Sanjaya.

Sorry to bother you with this.

While I am fairly certain that we had a brief discussion about Wink's ability to summarize unstructured text, I've lost track of where that discussion might live. I seem to recall that you also provided an example.

Do you have any sense for where that discussion might live and, if not, can you point me to examples of such summarization?

Thanks very much.

Cordially,

Paul

**Comment by sanjayaksaxena** [comment]
Hello @paul-bell 

There is a plan to add this capability to winkNLP; it is not available right now. You can easily build a naive summarization tool by shortlisting top few sentences who have maximum similarity with the others; and subsequently display them in their order of occurrence. For determining similarity, you can choose one of the [methods available in winkNLP](https://winkjs.org/wink-nlp/similarity.html).

Best
Sanjaya
